tests/test_vms_exist.py

```diff
@@ -36,6 +36,12 @@ def _check_kernel(self, vm):
         assert kernel_version.endswith("-grsec-workstation")
         assert kernel_version == EXPECTED_KERNEL_VERSION
 
+        u2mfn_filepath = "/usr/lib/modules/{}/updates/dkms/u2mfn.ko".format(EXPECTED_KERNEL_VERSION)
+        # cmd will raise exception if file not found
+        stdout, stderr = vm.run("sudo test -f {}".format(u2mfn_filepath))
+        assert stdout == ""
```

Comment:
Looks like adding `.decode("utf-8")` to stdout/stderr resolves the tests for me locally

Reply:
Thanks, you're right! Updated the string checks `""` -> `b""`, now passing for me.